packages/mobile/webpack.config.ts

```diff
@@ -121,7 +121,7 @@ const rendererPlugins = [
         filename: '[name].css',
     }),
     new DefinePlugin({
-        features: features,
+        features: JSON.stringify(features),
```

Comment:
Why do we want to string-ify this?

Reply:
Because otherwise the types do not match, we now have it typed on the other side too